wordle-web/src/App.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+  <header>
+    <img alt="Vue logo" class="logo" src="@/assets/logo.svg" width="125" height="125" />
+
+    <div class="wrapper">
+      <HelloWorld msg="You did it!" />
+
+      <nav>
+        <RouterLink to="/">Home</RouterLink>
+        <RouterLink to="/about">Class</RouterLink>
+        <RouterLink to="/about">About</RouterLink>
```

Comment:
Redundant links to the same page here, I would remove one of these for now.

Reply:
I'll keep them in just in case we end up using it later.